manim/grpc/gen/frameserver_pb2.py

```diff
@@ -1,743 +1,790 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: frameserver.proto
-
+"""Generated protocol buffer code."""
```

Comment:
This is generated one right? So it doesn't need review?

Reply:
That's right.